packages/akashify/deploy-akash.template.yml

```diff
@@ -0,0 +1,154 @@
+version: "2.0"
+
+services:
+  postgres:
+    image: noahsaso/subql-postgres:v1
+    expose:
+      - port: 5432
+        to:
+          - service: subquery-node
+          - service: graphql-engine
+          - service: cron
+    env:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD={{dbPassword}}
+    # uncomment when persistent storage works
+    # params:
+    #   storage:
+    #     data:
+    #       mount: /var/lib/postgresql/data
+
+  cron:
+    image: ghcr.io/ovrclk/akash-postgres-restore:v0.0.4
+    env:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD={{dbPassword}}
+      - POSTGRES_HOST=postgres
+      - POSTGRES_PORT=5432
+      - POSTGRES_DATABASE=postgres
+      - BACKUP_PATH={{backup.bucket}}/{{backup.folder}}
+      # get these from s3 bucket provider
+      - BACKUP_KEY={{backup.key}}
+      - BACKUP_SECRET={{backup.secret}}
+      - BACKUP_PASSPHRASE={{backup.encryptionPassphrase}}
+      - BACKUP_HOST={{backup.host}}
+      # every 15 minutes
+      - BACKUP_SCHEDULE={{backup.schedule}}
+      - BACKUP_RETAIN={{backup.retain}}
+    depends_on:
+      - service: postgres
+
+  subquery-node:
+    image: noahsaso/subql-node-cosmos:v1.10.2-7
+    expose:
+      - port: 3000
+        to:
+          - service: graphql-engine
+    depends_on:
+      - service: postgres
+      - service: cron
+    env:
+      - DB_USER=postgres
+      - DB_PASS={{dbPassword}}
+      - DB_DATABASE=postgres
+      - DB_HOST=postgres
+      - DB_PORT=5432
+      # zip must contain dist (from npm run codegen && npm run build),
+      # project.yaml, and schema.graphql
+      - PROJECT_ZIP_URL={{zipUrl}}
+
+  graphql-engine:
+    image: noahsaso/subql-query:v1.5.0-7
+    expose:
+      - port: 3000
+        to:
+          - global: true
+    depends_on:
+      - service: postgres
+      - service: subquery-node
+    env:
+      - DB_USER=postgres
+      - DB_PASS={{dbPassword}}
+      - DB_DATABASE=postgres
+      - DB_HOST=postgres
+      - DB_PORT=5432
+
+profiles:
+  compute:
+    postgres:
+      resources:
+        cpu:
+          units: 1
+        memory:
+          size: 1Gi
+        storage:
+          size: 1Gi
+        # replace above storage key with below when persistent storage works
+        # storage:
+        #   - size: 1Gi
+        #   - name: data
+        #     size: 5Gi
+        #     attributes:
+        #       persistent: true
+        #       class: beta2
+    cron:
+      resources:
+        cpu:
+          units: 1
+        memory:
+          size: 1Gi
+        storage:
+          size: 1Gi
+    subquery-node:
+      resources:
+        cpu:
+          units: 1
+        memory:
+          size: 1Gi
+        storage:
+          size: 1Gi
+    graphql-engine:
+      resources:
+        cpu:
+          units: 1
+        memory:
+          size: 1Gi
+        storage:
+          size: 1Gi
```

Comment:
I assume you will have to increase these to handle prod traffic (psql db especially will work better handling simultaneous client connections with more cores / ram.